pdns/pdnsutil.cc

```diff
@@ -933,6 +933,24 @@ int createZone(const DNSName &zone) {
     cerr<<"Domain '"<<zone.toString()<<"' was not created!"<<endl;
     return 1;
   }
+
+  DNSResourceRecord rr;
+  rr.qname = zone;
+  rr.auth = 1;
+  rr.ttl = ::arg().asNum("default-ttl");
+  rr.qtype = "SOA";
+  string soa = (boost::format("%s %s 1")
+    % ::arg()["default-soa-name"]
+    % (::arg().isEmpty("default-soa-mail") ? (DNSName("hostmaster.") + zone).toString() : ::arg()["default-soa-mail"])
```

Comment:
`toStringNoDot()` (see [docs](https://doc.powerdns.com/md/types/): `Warning: Host names and the MNAME of a SOA records are NEVER terminated with a '.' in PowerDNS storage!`). Although with `DNSName`, we don't care it is good form to do in a backwards compatible way.